[BUG] `bokeh init` creates a `package.json` which refers to the deprecated Bokeh JS node package

#### Software versions
bokeh==2.4.0
python==3.9.5
npm==7.21.0

#### Description of expected behavior and the observed behavior
Running `bokeh init` creates a `package.json` file that refers to the deprecated `bokehjs` Node package (https://www.npmjs.com/package/bokehjs). It should instead refer to the current `@bokeh/bokehjs` (https://www.npmjs.com/package/@bokeh/bokehjs) package.

#### Complete, minimal, self-contained example code that reproduces the issue

```
mkdir my_extension
cd my_extension
bokeh init
bokeh build
```
errors with:
```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for bokehjs@^2.4.0.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm install failed with exit code 1.
```

It is fixed by changing `bokehjs` to `@bokeh/bokehjs` in the `package.json` dependencies.

--- a/bokehjs/src/compiler/build.ts
+++ b/bokehjs/src/compiler/build.ts
@@ -144,7 +144,7 @@ export async function init(base_dir: Path, _bokehjs_dir: Path, base_setup: InitO
     keywords: [],
     repository: {},
     dependencies: {
-      bokehjs: `^${setup.bokehjs_version}`,
+      "@bokeh/bokehjs": `^${setup.bokehjs_version}`,
     },
     devDependencies: {},
   }

--- a/tests/unit/bokeh/test_ext.py
+++ b/tests/unit/bokeh/test_ext.py
@@ -35,7 +35,7 @@ def test_ext_commands(tmpdir) -> None:
 
     assert _names(tmp) == []
 
-    assert ext.init(tmp, bokehjs_version="1.3.4") is True
+    assert ext.init(tmp, bokehjs_version="2.4.0") is True
     assert _names(tmp) == [
         "bokeh.ext.json",
         "index.ts",

